calicoctl/commands/get.go

```diff
@@ -70,6 +70,9 @@ Description:
     * networkPolicy
     * globalNetworkPolicy
     * profile
+    * bgpConfiguration
+    * felixConfiguration
+    * clusterInformation
```

Comment:
given the size of this list now, I wonder if we should either (a) alphabetize this, or (b) just refer to the documentation.

perhaps you could do a quick poll and see what people think.

or maybe take a look at the kubectl help and see if that specifies the valid resource types.